src/components/header.jsx

```diff
@@ -21,16 +23,16 @@ const NavBar = () => {
         <Link to="/login" className="btn btn-primary mg-r-md">
           Signup
         </Link>
-        <Link to="/products" className="align-items-center fs-xl mg-x-md">
+        <Link to="/products" className="align-items-center fs-xl mg-x-md" title="Market">
         <i className="fas fa-store"></i>
         </Link>
-        <div className="badgeWrap mg-md">
+        <div className="badgeWrap mg-md" title="My Cart">
           <Link to="cart">
-            <div className="fs-xl fas fa-shopping-cart"></div>
+            <div className="fs-xl fas fa-shopping-cart" ></div>
           </Link>
-          <div className="fs-sm badge badgeTopRight badgeRed">8</div>
+          <div className="fs-sm badge badgeTopRight badgeRed">{cartState.state.CartItems.length>9?("9+"):cartState.state.CartItems.length}</div>
```

Comment:
This can be further simplified. Just send the state itself.

Reply:
i had the same name for state, so i willl refactor it later
because of the same name, i couldn't destructure it here, as it will conflict with wishlist state. thanks for pointing it out